sound/soc/sof/ipc4-topology.c

```diff
@@ -1485,6 +1485,85 @@ static int sof_ipc4_widget_free(struct snd_sof_dev *sdev, struct snd_sof_widget
 	return ret;
 }
 
+static int sof_ipc4_get_queue_id(struct snd_sof_widget *swidget, const char *peer_name,
```

Comment:
peer_name is not intuitive I think. Instead pass the source_widget and sink_widget and change to bool to say which widget you want the queue_id set for ie sink_widget or src_widget

Reply:
I prefer to target compared to peer_name